package.json

```diff
@@ -42,13 +43,13 @@
     "http-status-codes": "^2.3.0",
     "joi": "17.11.0",
     "jquery": "^3.7.1",
-    "jsonwebtoken": "^9.0.0",
+    "jsonwebtoken": "9.0.2",
     "jwk-to-pem": "^2.0.5",
     "mssql": "^10.0.2",
     "nunjucks": "^3.2.3",
     "pg-promise": "^11.5.5",
     "uuid": "^8.3.2",
-    "webdriverio": "^8.16.12"
+    "webdriverio": "9.4.2"
```

Comment:
Is this really a regular dependency, and not a devDependency? What on Earth are we using this for in the main code?

Reply:
aah hadn't even noticed that, just bumped because Snyk was moaning. I'll move it. 

Pretty sure non of the acceptance test stuff does anything anyway, but didn't want to go down that rabbit hole just now. Should maybe speak to @berlin-ee about cleaning it all up?

Reply:
done in latest commit